CHANGELOG.md

```diff
@@ -3,14 +3,57 @@
+- [fastawindows module](https://nf-co.re/modules/fastawindows) computes statistics across windows over the genome FASTA file.
+- Parameters for [diamond_blastp module](https://nf-co.re/modules/diamond_blastp) should be specified in `nextflow.config` file.
 
-### `Dependencies`
+This release marks the point where the pipeline was moved from Snakemake at [blobtoolkit/blobtoolkit](https://github.com/blobtoolkit/blobtoolkit) over to Nextflow DSL2 at [sanger-tol/blobtoolkit](https://github.com/sanger-tol/blobtoolkit). There are two subworkflows in the Snakemake version that are still being implemented in Nextflow: `diamond_blastx` and `blastn`.
 
-### `Deprecated`
+### Parameters
+
+| Old parameter | New parameter   |
+| ------------- | --------------- |
+|               | --accession     |
+|               | --blastp_outext |
+|               | --blastp_cols   |
+|               | --fasta         |
+|               | --input         |
+|               | --taxon         |
+|               | --yaml          |
+
+> **NB:** Parameter has been **updated** if both old and new parameter information is present. </br> **NB:** Parameter has been **added** if just the new parameter information
+> is present. </br> **NB:** Parameter has been **removed** if new parameter information isn't present.
+
+### Software dependencies
+
+Note, since the pipeline is using Nextflow DSL2, each process will be run with its own [Biocontainer](https://biocontainers.pro/#/registry). This means that on occasion it is e
+ntirely possible for the pipeline to be using different versions of the same tool. However, the overall software dependency changes compared to the last release have been liste
+d below for reference. Only `Docker` or `Singularity` containers are supported, `conda` is not supported.
+
+| Dependency             | Old version | New version |
+| ---------------------- | ----------- | ----------- |
+| blastn                 | 2.12.0      |             |
+| blobtools              | 4.0.7       | 4.1.2       |
+| busco                  | 5.3.2       | 5.4.3       |
+| fasta_windows          |             | 0.2.4       |
+| gunzip                 |             | 1.10        |
+| minimap2               | 2.24-r1122  |             |
+| multiqc                |             | 1.13        |
+| seqtk                  | 1.3-r106    |             |
+| windowmasker (blast\*) | 2.12.0      |             |
+
+> **NB:** Dependency has been **updated** if both old and new version information is present. </br> **NB:** Dependency has been **added** if just the new version information is
+> present. </br> **NB:** Dependency has been **removed** if version information isn't present.
+
+- blastn sub-workflow will be included in the next release.
+- blast\* version that runs windowmasker is specified, windowmasker is no longer used in this version.
```

Comment:
Is it because the new version of the pipeline expects a soft-masked assembly, whereas the Snakemake pipeline could do the masking itself ? That should be listed in the main list too.

Reply:
The snakemake version had windowmasker, we could bring it back in v2.0. The BLAST step is the one that really needs masked genome.